package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "arsenal",
-  "version": "1.1.0",
+  "version": "1.1.0-utapi",
```

Comment:
it's in a drop me commit to make sure integration was picking up the right version.